app/components/application_record_card/view.rb

```diff
@@ -23,7 +24,7 @@ def trainee_name
     def subject
       return "No subject provided" if record.subject.blank?
 
-      record.subject
+      subjects_for_summary_view(record.subject, record.subject_two, record.subject_three)
```

Comment:
Can you update https://rtt-review-pr-895.herokuapp.com/view_components/application_record_card/view/ to reflect the changes please

Reply:
added